library/src/main/java/com/novoda/downloadmanager/lib/DownloadsTables.java

```diff
@@ -0,0 +1,18 @@
+package com.novoda.downloadmanager.lib;
+
+final class DownloadsTables {
+
+    /**
+     * Name of downloadstable in the database
```

Comment:
Do we need these javadocs on these fields? I think the names of the variables are self explanatory.